src/hydra-queue-runner/build-remote.cc

```diff
@@ -340,28 +340,29 @@ void State::buildRemote(ref<Store> destStore,
                 case BuildResult::PermanentFailure:
                     result.stepStatus = bsFailed;
                     result.canCache = true;
-                    result.errorMsg = "";
                     break;
                 case BuildResult::InputRejected:
                 case BuildResult::OutputRejected:
                     result.stepStatus = bsFailed;
                     result.canCache = true;
+                    result.errorMsg += " {inp/out rejected}";
```

Comment:
Hm, it's not clear to me what this adds over the `stepStatus`, which is already stored in the database IIRC.

Reply:
Added stepStatus information to logging output (not to database for web ui output) and removed this text where not covered by the stepStatus.